derp/derp_server.go

```diff
@@ -838,6 +847,7 @@ func (s *Server) accept(ctx context.Context, nc Conn, brw *bufio.ReadWriter, rem
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	perClientSendQueueDepth := getPerClientSendQueueDepth()
```

Comment:
Can we move this to `Server` construction?

Reply:
Sure I just didn't want to call the function twice, but can stick it in there and call it just once.